_src-lp/blocks/b-productswithselectors/b-productswithselectors.js

```diff
@@ -22,11 +23,21 @@ export default function decorate(block) {
     // create the 2 selectors
     const labelName = selectorsName.split(',');
     // devices
-    const optionsDevices = Array(100).fill().map((_, d) => {
+    let devicesMin = 3;
```

Comment:
please write at least a todo comment on what should be tested via jest unit testing later on regarding this 30 logic

Reply:
Added the todo comment